core/src/main/java/se/tre/freki/core/MetaClient.java

```diff
@@ -198,13 +199,18 @@ public void deleteAnnotation(final Annotation note) {
    */
   public ListenableFuture<Boolean> update(final LabelMeta meta) {
     return transform(getLabelMeta(meta.type(), meta.identifier()),
-        new AsyncFunction<LabelMeta, Boolean>() {
+        new AsyncFunction<Optional<LabelMeta>, Boolean>() {
```

Comment:
We create the empty string as agreed previously. So the meta will never be "empty" or "missing" it will just be a string that does not contain anything.